### ISSUE TITLE
Material Pass pointAttenuation not working as expected

### ISSUE THREAD

**Issue by Niproblema** [OGRECave/ogre#3086]
#### System Information
- Ogre Version: :grey_question: 14.2
- Operating System / Platform: :grey_question:  ubuntu20.04
- RenderSystem: :grey_question: OpenGL

#### Detailled description

First of all `setPointMinSize(1.0f)` and  `setPointMaxSize(2.0f)` are not taken into account anywhere. Attenuation comes up with arbitrary value that is not clamped

Setting `setPointAttenuation(true, 1, 0, 0)` results in huge points, even though it should be equal to disabled
 
setting any other value is also pointless as it randomly increases point size with distance.

**Comment by Niproblema** [comment]
How may I use that to fix my attenuation 🤔

**Comment by paroj** [comment]
you can place a breakpoint in the above function and check whether all values make sense

**Comment by Niproblema** [comment]
I get values minSize = 98 and maxSize = 980, which is completely wrong

**Comment by paroj** [comment]
is that before or after this line?
```cpp
minSize = minSize * mActiveViewport->getActualHeight();
```

**Comment by paroj** [comment]
then the values are plausible, as they were multiplied by your screen height.

If you want to have minSize = 1px, dont setPointMinSize(1), but rather setPointMinSize(1.0/1080), if you are at 1920x1080.

**Comment by Niproblema** [comment]
Ah you are correct, I managed to get it working.. kind of. I find attenuation parameters extremely confusing. 
setting `setPointAttenuation(true, 1.f, 500.f, 500.f);` seems to be a decent setting for my scene. 
Looking at the equation 
`attenuation=1/(constant+linear∗dist+quadratic∗d^2)` I have no idea why such parameters might work.
 I also noticed that enabling FSAA turns my billboard sprites into dots on my Intel VGA machine and suprisingly to solid polygons on an AMD VGA.
